### ISSUE TITLE
Option to disable auto selection of parent category if child is selected and retain default woocommerce behavior 

### ISSUE THREAD

**Issue by cristaves** [weDevsOfficial/dokan#1710]
Dokan changed the default category behavior of woocommerce. When a product is given a child category, the parent category of that child is automatically checked. This setting interferes with woocommerce's Related Products selection and also bloats product pages for products with multiple categories as explained in this ticket [Ticket](https://wordpress.org/support/topic/parent-category-automatically-selected-with-child-product-category-2/)  

Woocommerce already provides for the checking of both the child and parent for those who desire it. Woocommerce behavior also allows the site owner to choose to either select both parent and child or not on per product basis which is also good.

A good idea would be to provide a setting in the dokan settings to enable this automatic checking of the parent or otherwise retain the default woocommerce behavior

**Comment by Dkmecik** [comment]
I came here to request the same thing. This creates a big problem when you create a coupon that excludes products in a specific child category, but because dokan also added them to a parent category they are still included in the coupon.

Looking forward to dumping dokan as soon as our custom site is completed. this just adds yet another reason!

**Comment by cristaves** [comment]
Its a serious bug but dokan considers it as a feature. I also realised its conflicting with Google ads and listings plugin. I could exclude specific child categories from serving products in google shopping ads but now I cannot if i want products from another child category with the same parent to serve. I had to waste a lot of money on ad clicks to products in subcategories I had excluded from serving. I disabled dokan and everything is now fine. I am currently studying other dokan alternatives.
